packages/dashboard/app/withdraw/utils/transactionService.ts

```diff
@@ -0,0 +1,188 @@
+import { ethers } from 'ethers';
+import { PKPEthersWallet } from '@lit-protocol/pkp-ethers';
+
+interface TokenDetails {
+  address: string;
+  symbol: string;
+  decimals: number;
+}
+
+interface TransactionResult {
+  success: boolean;
+  hash: string;
+  error?: string;
+  receipt?: ethers.providers.TransactionReceipt;
+}
+
+interface SendTransactionWithRetryParams {
+  pkpWallet: PKPEthersWallet;
+  txOptions: ethers.providers.TransactionRequest;
+  provider: ethers.providers.JsonRpcProvider;
+}
+
+interface SendEthTransactionParams {
+  pkpWallet: PKPEthersWallet;
+  amount: ethers.BigNumber;
+  recipientAddress: string;
+  provider: ethers.providers.JsonRpcProvider;
+}
+
+interface SendTokenTransactionParams {
+  pkpWallet: PKPEthersWallet;
+  tokenDetails: TokenDetails;
+  amount: ethers.BigNumber;
+  recipientAddress: string;
+  provider: ethers.providers.JsonRpcProvider;
+}
+
+/**
+ * Helper function to send a transaction with automatic retry on "replacement fee too low" error
+ */
+async function sendTransaction({
+  pkpWallet,
+  txOptions,
+  provider,
+}: SendTransactionWithRetryParams): Promise<TransactionResult> {
+  try {
+    const tx = await pkpWallet.sendTransaction(txOptions);
+    const receipt = await provider.waitForTransaction(tx.hash, 1);
+
+    return {
+      success: receipt.status === 1,
+      hash: tx.hash,
+      receipt
+    };
+  } catch (error: any) {
+    if (error.message && error.message.includes('insufficient funds for intrinsic transaction cost')) {
+      return {
+        success: false,
+        hash: '',
+        error: 'Insufficient funds for transaction. Please add more ETH to your wallet.'
+      };
+    } else {
+      return {
+        success: false,
+        hash: '',
+        error: error.message || 'Failed to send transaction'
+      };
+    }
+  }
+}
+
+/**
+ * Sends an ETH transfer transaction
+ */
+export async function sendEthTransaction({
+  pkpWallet,
+  amount,
+  recipientAddress,
+  provider
+}: SendEthTransactionParams): Promise<TransactionResult> {
+  try {
+    const ethBalance = await pkpWallet.getBalance();
+    if (amount.gt(ethBalance)) {
+      return {
+        success: false,
+        hash: '',
+        error: `Insufficient balance to withdraw specified amount and pay for current gas fees.`
+      };
+    }
+
+    const txOptions = {
+      to: recipientAddress,
+      value: amount
+    };
+
+    return await sendTransaction({
+      pkpWallet,
+      txOptions,
+      provider,
+    });
+  } catch (error: any) {
+    console.error('Error sending ETH transaction:', error);
+    return {
+      success: false,
+      hash: '',
+      error: error.message || 'Failed to send ETH'
```

Comment:
```suggestion
      error: error.message || 'Failed to send native asset'
```

Because we are multichain now we cannot talk about Eth anymore. This should at least be changed in the user facing strings